android/java/org/chromium/chrome/browser/crypto_wallet/util/Utils.java

```diff
@@ -149,10 +159,22 @@ public static String getTextFromClipboard(Context context) {
             return pasteData;
         } else if (!(clipboard.getPrimaryClipDescription().hasMimeType(MIMETYPE_TEXT_PLAIN))) {
             return pasteData;
-        } else {
-            ClipData.Item item = clipboard.getPrimaryClip().getItemAt(0);
-            return item.getText().toString();
         }
+        ClipData.Item item = clipboard.getPrimaryClip().getItemAt(0);
+
+        return item.getText().toString();
+    }
+
+    public static void clearClipboard(String textToCompare, int delay) {
```

Comment:
in several cases above caller uses `delay==0`, so maybe for such cases it worth not to use the `Timer`, but call the cleaning procedure immediately?

Reply:
it executes the task in the background thread. I don't see any issues with executed it right away when we pass 0. What are the cons of that?

Reply:
it's cleaner code even as to me instead of putting some if conditions

Reply:
only disadvantage is just creating of two extra unneeded objects, `new Timer()` and `new TimerTask()`

Reply:
it happens so rare, I don't think there are any issues with that.